src/application/KeyboardShortcutsModal.tsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from 'react';
+
+import { Modal } from '../components/Modal';
+import { useListenEvent } from '../hooks/useListenEvent';
+import { cx } from '../lib/cx';
+
+interface ModalTab {
+  name: string;
+  disabled?: boolean;
+  groups: {
+    header: string;
+    shortcuts: {
+      label: string;
+      keys: string[];
+    }[];
+  }[];
+}
+
+const MODAL_TABS: ModalTab[] = [
+  {
+    name: 'Essentials',
+    groups: [
+      {
+        header: 'File',
+        shortcuts: [
+          makeShortcut('Save File', ['⌘', 'S']), //
+          makeShortcut('New File', ['⌘', 'N']),
+        ],
+      },
+      {
+        header: 'Quick Panes',
+        shortcuts: [
+          makeShortcut('Quick Actions', ['⌘', 'K']), //
+          makeShortcut('Quick Files', ['⌘', 'P']),
+        ],
+      },
+      {
+        header: 'AI Tools',
+        shortcuts: [
+          makeShortcut('AI Text Completion', ['⌘', 'J']), //
+        ],
+      },
+    ],
+  },
+  {
+    name: 'Editor',
+    disabled: true,
+    groups: [],
+  },
+  {
+    name: 'AI Tools',
+    disabled: true,
+    groups: [],
+  },
+  {
+    name: 'File',
+    disabled: true,
+    groups: [],
+  },
+  {
+    name: 'References',
+    disabled: true,
+    groups: [],
+  },
+  {
+    name: 'View',
+    disabled: true,
+    groups: [],
+  },
+];
+
+function makeShortcut(label: string, keys: string[]) {
+  return {
+    label,
+    keys,
+  };
+}
+
+export function KeyboardShortcutsModal() {
+  const [isVisible, setVisible] = useState(false);
+  const [selectedTab, setSelectedTab] = useState(MODAL_TABS[0]);
+
+  useListenEvent('refstudio://menu/view/keyboard-shortcuts', () => setVisible(true));
+
+  return (
+    <Modal
+      className="hx-[568px] max-h-full w-[730px] max-w-[630px] bg-white text-modal-txt-primary"
+      open={isVisible}
+      onClose={() => setVisible(false)}
+    >
+      <div className="flex w-full flex-col gap-1">
+        <Tabs selectedTab={selectedTab} onClick={setSelectedTab} />
+        <KeysTabPane selectedTab={selectedTab} />
+      </div>
+    </Modal>
+  );
+}
+
+function Tabs({ selectedTab, onClick }: { selectedTab: ModalTab; onClick: (tab: ModalTab) => void }) {
+  return (
+    <div>
+      <div className="relative flex w-full pl-3 pt-1">
+        {MODAL_TABS.map((tab) => (
+          <div
+            className={cx('px-4 py-3', {
+              'font-bold': selectedTab.name === tab.name,
+              'z-[1] border border-b border-modal-border border-b-white': selectedTab.name === tab.name,
+              'cursor-pointer hover:font-bold': selectedTab.name !== tab.name,
```

Comment:
This is just a design preference, but I would keep the cursor-pointer even for the active tab

And a nit comment: in the design the tab borders are rounded and there is no top margin (the left and right border take the whole height and there is no top border. I don't mind the top border, but I think we should make everything rounded for consistency with the rest of the app, especially with the editor tabs.

Reply:
Thanks for your feedback. We redesigned this screen so now we don't need tabs and round corners  🎉 

<img width="1052" alt="image" src="https://github.com/refstudio/refstudio/assets/174127/e1a84e18-3e45-4b43-b63b-80c6de5790ae">